From f39e6e220b9ca1ebd3a27d19056d2935838a3a38 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 21 Jan 2019 16:58:10 -0700
Subject: [PATCH 07/14] Revert "msm8974-common: sepolicy: Label sysfs_net,
 resolve denials"

This reverts commit 97ff0e6d32e0fb32caf84ccfaff3ea0fe8ea869a.

Change-Id: Ib609a1a9987598be26e2fe32cc77ea9f57c9c63d
---
 sepolicy/common/file_contexts               | 4 ----
 sepolicy/common/hal_wifi_hostapd_default.te | 1 -
 2 files changed, 5 deletions(-)

diff --git a/sepolicy/common/file_contexts b/sepolicy/common/file_contexts
index a5cfa80..f247e78 100644
--- a/sepolicy/common/file_contexts
+++ b/sepolicy/common/file_contexts
@@ -72,10 +72,6 @@
 # sysfs - mdnie
 /sys/devices/virtual/mdnie/mdnie(/.*)?                  u:object_r:sysfs_mdnie:s0
 
-# sysfs - net
-/sys/devices/msm_sdcc\.[0-9]/mmc_host/.*/net(/.*)?      u:object_r:sysfs_net:s0
-/sys/devices/virtual/net(/.*)?                          u:object_r:sysfs_net:s0
-
 # sysfs - sec
 /sys/devices/platform/sec-thermistor(/.*)?              u:object_r:sysfs_sec_thermistor:s0
 /sys/devices/virtual/sec/bamdmux(/.*)?                  u:object_r:sysfs_sec_bamdmux:s0
diff --git a/sepolicy/common/hal_wifi_hostapd_default.te b/sepolicy/common/hal_wifi_hostapd_default.te
index 5a7a60d..e71f109 100644
--- a/sepolicy/common/hal_wifi_hostapd_default.te
+++ b/sepolicy/common/hal_wifi_hostapd_default.te
@@ -1,2 +1 @@
-allow hal_wifi_hostapd_default sysfs_net:lnk_file { getattr read };
 allow hal_wifi_hostapd_default wlan_device:chr_file read;
-- 
2.17.1

